invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/api/DepositRecordSerializer.js

```diff
@@ -408,7 +412,12 @@ export class RDMDepositRecordSerializer extends DepositRecordSerializer {
     ]);
 
     // Save pids so they are not removed when an empty value is passed
-    let savedPIDsFieldValue = originalRecord.pids || {};
+    const savedNoDOIPIDS = this.originalPublishedRecord.pids
```

Comment:
save any already added published PID that is not part of the form. From the form we reset the pids field so we can e.g "unreserve" an externally managed DOI (it is just a metadata update).